src/matter/cluster/BasicCluster.ts

```diff
@@ -34,6 +40,16 @@ export const BasicCluster = Cluster(
         localConfigDisabled: OptionalWritableAttribute(16, BooleanT, false), /* writeAcl: manage */
         reachable: OptionalAttribute(17, BooleanT, true),
         uniqueId: OptionalAttribute(18, StringT), /* maxLength: 32 */
+        capabilityMinima: Attribute(19, CapabilityMinimaT),
     },
     {},
 );
+
+/*
+Events:
```

Comment:
SHould I describe events like here?

Reply:
Not sure it is necessary at this point since there are not supported.

Or you can just copy / paste the full event specs from the XML file in the comment: I don't know how they will structured here and just the information below are not enough (optional, priority, type...)

Reply:
Ok, will add XMLs. Good idea for later reference